core/env/sentinel.go

```diff
@@ -43,10 +47,34 @@ func SentinelInitCommandPath() string {
 // Returns:
 //
 //	string: The path to the Sentinel initialization command tombstone.
-func SentinelInitCommandTombstonePath() string {
+func InitCommandTombstonePathForSentinel() string {
 	p := os.Getenv("VSECM_SENTINEL_INIT_COMMAND_TOMBSTONE_PATH")
 	if p == "" {
 		p = "/opt/vsecm-sentinel/tombstone/init"
 	}
 	return p
 }
+
+// InitCommandRunnerWaitTimeoutForSentinel initializes and returns the timeout
+// duration for waiting for Sentinel to acquire an SVID.
+//
+// If the environment variable "VSECM_SENTINEL_INIT_COMMAND_RUNNER_WAIT_TIMEOUT"
+// is set and valid, it uses the value provided by the environment variable as
+// the timeout duration.
+// If the environment variable is not set or invalid, a default timeout of
+// 300,000 milliseconds (5 minutes)
+//
+// Returns:
+//
+//	time.Duration: The max time duration that the Sentinel wiil wait for an SVID.
+func InitCommandRunnerWaitTimeoutForSentinel() time.Duration {
+	p := os.Getenv("VSECM_SENTINEL_INIT_COMMAND_RUNNER_WAIT_TIMEOUT")
+	if p == "" {
+		p = "300000"
```

Comment:
300 secs is the default timeout for Sentinel to way.

It typically takes 30 seconds for everything to reconcile, so 300 secs is more than graceful of a wait time.